### ISSUE TITLE
Reply messages are sometimes not displayed

### ISSUE THREAD

**Issue by SakulFlee** [element-hq/element-android#8162]
### Steps to reproduce

1. Setup a standard Home-Server (Synapse) and [Mautrix-Discord](https://github.com/mautrix/discord).
2. Start a chat with someone on Discord, make them **reply** to your messages.
3. View the reply message on Element-Android.

#### What did you expect?

On Web and Desktop, the message is rendered correctly and can be read. This is to be expected on the Android side too.

#### What happened instead?

The reply message appears blank and can't be read.

I've detailed and made a demonstration over on [Mautrix-Discord's repository](https://github.com/mautrix/discord), which can be viewed [here (#59)](https://github.com/mautrix/discord/issues/59).

What I found so far is that inside the source of the reply over on Matrix the `content.format` and `content.formatted_body` fields are missing. Which I am guessing the Android Client expects (but Web and Desktop doesn't?).
The message is there, in the source view, and when replying to the "empty reply" it also shows up in the preview above the input field, but it won't display inside the chat.

I've also filed a bug inside the App with logs.
I'll happily provide more information and, if possible, fix this myself if I know where to start :)

### Your phone model

Samsung A13

### Operating system version

Android 13

### Application version and app store

Element version 1.5.25 [40105251] (G-f74a885a), Matrix SDK 1.5.25 (f74a885a), OLM version 3.2.12 from Google PlayStore

### Homeserver

Selfhosted Synapse server version 1.71.0-r0; hosted on matrix.sakul-flee.de, federated

### Will you send logs?

Yes

### Are you willing to provide a PR?

Yes

**Comment by noClaps** [comment]
This happens to me in non-bridged rooms too.

**Comment by Sakul6499** [comment]
Interesting, this suggests it not being related to the bridge but some issue on the Android-App and/or Matrix server.

@noClaps Would you be so kind to share relevant information like the "source" of those messages, which homeserver, version of homeserver, etc.?

**Comment by noClaps** [comment]
I'm on the matrix.org homeserver, which I'm assuming is on the latest version.

I definitely think this is a client issue, as I don't experience this on other Matrix clients like Fluffychat.

When replying with `/me message`:
![Screenshot_20230305-003120~3](https://user-images.githubusercontent.com/103803129/222935378-90079f55-d2ba-40cd-918a-d09c6b061a00.png)

![Screenshot_20230305-003504~2](https://user-images.githubusercontent.com/103803129/222935295-2cfe30bb-48fc-4526-af6d-a2ff975efd15.png)

When replying with image:
![Screenshot_20230305-003544~4](https://user-images.githubusercontent.com/103803129/222935383-0c753ddf-084e-4d1c-a5d1-5dd7ac73137a.png)

![Screenshot_20230305-003839](https://user-images.githubusercontent.com/103803129/222935328-e83bbfa0-8b38-4b4e-821c-f528307ebdd4.png)

I'm not sure if this is what you're looking for when you said "source". If it's something else, please let me know.

**Comment by Sakul6499** [comment]
@noClaps That's exactly what I wanted to see, thank you!

I believe this also confirms my suspicion that, _for some reason_, if the `format` field is set to `org.matrix.custom.html` (or empty), the Android App fails to display it properly.

**Comment by noClaps** [comment]
It also seems to fail if the reply is an image, which doesn't have the `format` field at all.

**Comment by wapsi** [comment]
I'm having this issue as well. Especially the images do not show up in the Element Android version (on Android 11), when the user adds and image into reply message of existing message. In the desktop/web version of the Element this works just fine, though.

**Comment by Vrozaksen** [comment]
Same issue here!
Samsung S22 Ultra with Android 13

Element:
Version: 1.5.32 [40105322] (G-8dcb5f70)
Matrix SDK: 1.5.32 (8dcb5f70)

**Comment by tduboys** [comment]
Hello,
Same issue using signal bridge (issue https://github.com/mautrix/signal/issues/384 )

Element version 1.6.10

**Comment by cmbarba** [comment]
I'm experiencing this consistently with every reply in bridged rooms. My own replies are displayed, but replies from other bridged users are not. Replies are only missing from the room views; they are visible in the room preview in the list of rooms.

**Comment by wapsi** [comment]
FYI: As a workaround I'm using SchildiChat now (https://schildi.chat/), which is Element fork with some extra features/fixes applied. For instance, this issue does not happen on SchildiChat at all :)

**Comment by hamishmb** [comment]
I'm having this issue still, on Android 15 on OnePlus 6T, not with bridges, but with replies that include links.

**Comment by PatMulligan** [comment]
yesterday I installed Element Desktop on linux using

```
sudo apt install -y wget apt-transport-https
‍
sudo wget -O /usr/share/keyrings/element-io-archive-keyring.gpg https://packages.element.io/debian/element-io-archive-keyring.gpg
‍
echo "deb [signed-by=/usr/share/keyrings/element-io-archive-keyring.gpg] https://packages.element.io/debian/ default main" | sudo tee /etc/apt/sources.list.d/element-io.list

sudo apt update

sudo apt install element-desktop
```

With the **desktop** client, the android client is not seeing the reply content. However, I am quite sure the previous Flatpak installation, the replies came through fine

![image](https://github.com/user-attachments/assets/30448a47-6ec2-4fa6-a64f-999b46b277c4)

### Test 1 Decrypted Event Source (sent from linux desktop client)
![image](https://github.com/user-attachments/assets/ae410aab-2b2c-419f-b98f-f401f2bd11ad)


### test2 Decrypted event source (sent from android client)
![image](https://github.com/user-attachments/assets/92ba5176-41ac-4c29-b049-eb717dbb3887)

**Comment by ggogel** [comment]
I get this issue in some larger non-encrypted public rooms, while smaller encrypted rooms or 1:1 rooms are not affected at all. Only when I long press the reply I can read its content.

**Comment by PatMulligan** [comment]
> I get this issue in some larger non-encrypted public rooms, while smaller encrypted rooms or 1:1 rooms are not affected at all. Only when I long press the reply I can read its content.

The example I gave above is in a 1:1 encrypted room

**Comment by pcroland** [comment]
Issue should be reopened, this is still a problem.

**Comment by AppTester7** [comment]
Same problem here

Android Version: 12
Android Element Version: 1.6.24
I also tryed the beta - same thing

**Comment by hamishmb** [comment]
For the record, I'm running Android 15 with 1.6.24

**Comment by lethargosapatheia** [comment]
I've also had this problem for quite a while. I'm running Android 14 and element version 1.6.24. Initially this seemed to happen when someone replied to messages and then edited their own message, but I see it often happens also when they simply reply to a message. Really annoying. If the problem is fixed in 1.6.26, why isn't it publicly available?
